packages/manager-components/src/components/action-banner/action-banner.component.tsx

```diff
@@ -10,18 +10,25 @@ import {
   ODS_THEME_TYPOGRAPHY_SIZE,
 } from '@ovhcloud/ods-common-theming';
 
-export interface ActionBannerProps {
+export type ActionBannerProps = {
   message: string;
   cta: string;
+  type?: ODS_MESSAGE_TYPE;
   onClick: () => void;
-}
+};
 
-export function ActionBanner({ message, cta, onClick }: ActionBannerProps) {
+export function ActionBanner({
+  message,
+  cta,
+  type = ODS_MESSAGE_TYPE.info,
+  onClick,
+}: Readonly<ActionBannerProps>) {
   return (
     <OsdsMessage
-      type={ODS_MESSAGE_TYPE.warning}
-      color={ODS_THEME_COLOR_INTENT.warning}
+      type={type}
+      color={(type as unknown) as ODS_THEME_COLOR_INTENT}
```

Comment:
To be safer we could make an object :
```
const colorByType = {
  [ODS_MESSAGE_TYPE.error]: ODS_THEME_COLOR_INTENT.error,
  etc.
};
```

If there is a typo or they change the string behind it, it will break